packages/flutter/test/cupertino/app_test.dart

```diff
@@ -208,6 +208,26 @@ void main() {
     expect(scrollBehavior.runtimeType, MockScrollBehavior);
     expect(scrollBehavior.getScrollPhysics(capturedContext).runtimeType, NeverScrollableScrollPhysics);
   });
+
+  testWidgets('When `useExistingMediaQuery` is true an existing MediaQuery is used if one is available', (WidgetTester tester) async {
```

Comment:
Nit: Update these test titles to reflect the new name

Reply:
Sure, done ✅